README.md

```diff
@@ -85,6 +90,18 @@ ALTER TABLE ['{table_name}'] ALTER COLUMN ['{primary_key_column_name}'] int 
 ALTER TABLE ['{table_name}'] ADD CONSTRAINT PKey PRIMARY KEY CLUSTERED (['{primary_key_column_name}']);
 ```
 
+3. If you plan to use the trigger support, you need to enable [change tracking](https://docs.microsoft.com/sql/relational-databases/track-changes/about-change-tracking-sql-server) on the SQL database and the SQL table. Please note that enabling change tracking will add to the cost of the SQL server.
+
+```sql
+ALTER DATABASE ['your database name']
+SET CHANGE_TRACKING = ON
+(CHANGE_RETENTION = 2 DAYS, AUTO_CLEANUP = ON)
+
+ALTER TABLE ['your table name']
+ENABLE CHANGE_TRACKING
+WITH (TRACK_COLUMNS_UPDATED = ON)
```

Comment:
Since we are returning all columns for an update operation and not using SYS_CHANGE_COLUMNS, TRACK_COLUMNS_UPDATED does not need to be on right?

Reply:
There's more information about that below, but it just says that this functionality isn't currently used by the trigger. Because of that I think we should probably leave this out of the samples/docs so users aren't needlessly enabling it.

Reply:
Removed `TRACK_COLUMNS_UPDATED` option from Readme and code samples.